src/ci/azure-pipelines/steps/install-windows-build-deps.yml

```diff
@@ -18,9 +18,9 @@ steps:
 # one is MSI installers and one is EXE, but they're not used so frequently at
 # this point anyway so perhaps it's a wash!
 - script: |
-    powershell -Command "$ProgressPreference = 'SilentlyContinue'; iwr -outf is-install.exe https://rust-lang-ci-mirrors.s3-us-west-1.amazonaws.com/rustc/2017-08-22-is.exe"
-    is-install.exe /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-
     echo ##vso[task.prependpath]C:\Program Files (x86)\Inno Setup 5
+    curl.exe -o is-install.exe https://rust-lang-ci-mirrors.s3-us-west-1.amazonaws.com/rustc/2017-08-22-is.exe
+    is-install.exe /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-
```

Comment:
Hm, Is there a reason we're moving this after the `echo` statement? It looks like these two might be connected

Reply:
Ah yeah I couldn't figure out how to get this working with `bash` and AFAIK the exit code of the script is the last statement, so the thing which may fail is listed here last and the `echo`, which always succeeds, was moved above it.